src/CISDSC/CISDSC.psd1

```diff
@@ -4,7 +4,7 @@
 RootModule = 'CISDSC.psm1'
 
 # Version number of this module.
-ModuleVersion = '3.0.0'
+ModuleVersion = '3.1.1'
```

Comment:
Is this the correct increment?

Reply:
Yes, last time we merged the change we forgot to update it to 3.1.0 despite that being what the changelog states. So this would be 3.1.1 since it's a small change to the 3.1.0 version we released when adding Server 2022